obspy/scripts/runtests.py

```diff
@@ -81,27 +96,28 @@ def main():
     If profiling is enabled we disable interactivity as it would wait for user
     input and influence the statistics. However the -r option still works.
     """
-    _ensure_tests_requirements_installed()
-
-    import pytest
-    from pytest_jsonreport.plugin import JSONReport
-
-    report = (True if '--report' in sys.argv else
-              False if '--no-report' in sys.argv else None)
     if '-h' in sys.argv or '--help' in sys.argv:
         print(__doc__)
         sys.exit(0)
-    elif all(['--json-report-file' not in arg for arg in sys.argv]):
-        sys.argv.append('--json-report-file=none')
-    # Use default traceback for nicer report display.
-    sys.argv.append("--tb=native")
+    try:
+        import pytest
+        from pytest_jsonreport.plugin import JSONReport
+    except ImportError:
+        _ensure_tests_requirements_installed()
```

Comment:
This kinda works for now, but it effectively short-circuits checks for any other test requirements we may want to use in the future. Are you doing this to speed up test execution?